### ISSUE TITLE
Transfer learning

### ISSUE THREAD

**Issue by chichivica** [AlexeyAB/darknet#182]
Hi! 
Many thanks for YOLO for windows.

I trained my model to detect 2 classes. It worked great, but at some moment, a new **slightly** different images, appeared and the model began to make mistakes.
So, I collected comprehensive dataset of such "alien" for my model images. 
And now, I need to "continue" learning, **without** complete retraining. 
As I could found out that process named as "transfer learning", and I tried several times with no success, to fix my model. And therefore my question is: 

Is that that feasible?

I trained classifier model darknet19_448.cfg with standard params:
learning_rate=0.001
max_batches=100000

**Comment by AlexeyAB** [comment]
Hi,
> I trained my model to **detect** 2 classes.

> I trained **classifier** model darknet19_448.cfg

1. So, do you use Detector or Classifier?

2. Yes, transfer learning (network adaptation) is possible, if you haven't enought time to train from the beginning, then just add "alien" images, where aren't required objects, add them to train.txt and create for each empty label txt-file - for this you can use https://github.com/AlexeyAB/Yolo_mark - just pass through all images by pressing SPACE - for all images without objects empty txt-labels will be created. And contionue training with (old_images+new_images).
But the best way is to train from the beginning using (old_images+new_images).
